src/cmd.rs

```diff
@@ -355,8 +355,14 @@ impl Pipeline {
     /// Creates an empty pipeline.  For consistency with the `cmd`
     /// api a `pipe` function is provided as alias.
     pub fn new() -> Pipeline {
+        Self::with_capacity(0)
+    }
+
+    /// Creates an empty pipeline with pre-allocated capacity.  For consistency with the `cmd`
+    /// api a `pipe` function is provided as alias.
```

Comment:
The second part here is not true. The `pipe` function is only equivalent to `new()` above.

Reply:
oopss. my bad. will fix.